python/cuml/cuml/svm/svm_base.pyx

```diff
@@ -614,3 +650,7 @@ class SVMBase(Base,
         self.handle.sync()
 
         return out
+
+
+# Add TotalIters to the SVC/SVR class for easier access
+SVMBase.TotalIters = TotalIters
```

Comment:
nit/curiosity: why not just declare it within the class?

Reply:
Our magical base class tries to interpret it as a method for decorating and gets mad. I want to rip that thing out so bad.